tomcat/src/main/java/org/apache/coyote/http11/request/HttpRequest.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.coyote.http11.request;
+
+public class HttpRequest {
+
+    final private String requestMethod;
```

Comment:
```suggestion
    private final String requestMethod;
```
순서가 잘못 된 것 같네요 :)

1. Annotations
2. 접근자 (public/protected/private)
3. abstract
4. static
5. final
6. transient
7. volatile
8. default
9. synchronized
10. native
11. strictfp

Reply:
헉 ㅎ😱